File: 1 - BEGINNER/1002 - Area of a Circle/areaCircle.cs
using System;

class URI
{

  static void Main(string[] args)
  {

    double PI, raio, area;

    PI = 3.14159;
    raio = Convert.ToDouble(Console.ReadLine());

    area = PI * Math.Pow(raio, 2);

    Console.WriteLine($"A={area:F4}");

  }

}

File: 1 - BEGINNER/1002 - Area of a Circle/areaCircle.js
const input = require('fs').readFileSync('../../stdin', 'utf8');
const lines = input.split('\n');

const PI = 3.14159;
const raio = Number(lines.shift());

const area = PI * Math.pow(raio, 2);

console.log(`A=${area.toFixed(4)}`);